manifests/plugin/remote_execution/ssh.pp

```diff
@@ -63,4 +63,17 @@
       creates => $ssh_identity_path,
     }
   }
+
+  case $::osfamily {
+    'Debian': {
+      $core_package = 'ruby-smart-proxy-remote-execution-ssh-core'
+    }
+    default: {
+      $core_package = 'tfm-rubygem-smart_proxy_remote_execution_ssh_core'
+    }
+  }
+
+  package { $core_package:
+    ensure => installed',
+  }
```

Comment:
@iNecas I think that's a fair point.

@stbenjam in that case I'd build it (in this class) as follows:
```puppet
if $::osfamily == 'RedHat' and $::operatingsystem != 'Fedora' {
  $scl_prefix = 'tfm-'
} else {
  $scl_prefix = ''
}

foreman_proxy::plugin { 'remote_execution_ssh_core':
  package => "${scl_prefix}${::foreman_proxy::plugin_prefix}remote_execution_ssh_core",
}
```

Though I think this still pretends a bit too much that it is a plugin, it avoids duplication.